js/src/components/modals/SEMrushCountrySelector.js

```diff
@@ -165,7 +165,7 @@ class SEMrushCountrySelector extends Component {
 	 */
 	componentDidMount() {
 		//	Fire a new request when the modal is first opened
```

Comment:
This comment should be updated to reflect the new logic (will do as part of the CR).